common/models/config-file.json

```diff
@@ -1,5 +1,6 @@
 {
   "validateUpsert": true,
+  "base": "WorkspaceEntity",
```

Comment:
I am not sure whether ConfigFile should be extending WorkspaceEntity. IIIRC, in LoopBack 2.x, models were by default inheriting from a base class that become `PersistedModel` in LB 3.x, and this is the default behavior in 3.x. 

I believe no change should be necessary in this model definition file. Please revert this line.